pom.xml

```diff
@@ -866,7 +866,6 @@
           <source>${javac.target}</source>
           <target>${javac.target}</target>
           <compilerArgs>
-            <compilerArg>-Werror</compilerArg>
```

Comment:
why do you need this change?  This is a build-wide setting.

If netty upgrade resulted in some deprecation warnings you need to fix them

Reply:
Yeal, I try to restore this configuration and update the dependency license files. And now I get a problem about `TestTLS`.